deploy/crds/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -145,7 +145,7 @@ spec:
           description: PerformanceProfileStatus defines the observed state of PerformanceProfile.
           properties:
             conditions:
-              description: conditions represents the latest available observations
+              description: Conditions represents the latest available observations
```

Comment:
Not for this patch, Under the 7 rules of commits these kind of changes, even trivial, should be in a separate patch, so we can concentrate on the patch with the "real" changes.

Reply:
done